### PR TITLE
Feature/dst 181 apprenticeship standards

### PR DESCRIPTION
**Pull request by tomtollafield**
- Could change Type ("Standard" or "Framework") to a dropdown or content picker?

### CODE REVIEW

File: DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (lazcool) [inline DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json]: This file is needed for provisioning the site in the release pipeline.

File: DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json [hunk 2]
```diff
@@ -1507,6 +1590,115 @@
...
+                        {
+                            "FieldName": "NumericField",
+                            "Name": "MaximumFunding",
+                            "Settings": {
+                                "ContentPartFieldSettings": {
+                                    "DisplayName": "Maximum Funding Amount (£)",
+                                    "Position": "1"
+                                },
+                                "NumericFieldSettings": {
+                                    "Scale": 1
```
Comment (lazcool) [inline DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json]: Seems odd to store £ amounts to 1 decimal place. Either whole number or 2 decimal places makes more sense intuitively.

Comment (tomtollafield) [reply DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json]: They are whole pounds so I can change this to whole number

File: DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json (hunk 1 above)
Comment (tomtollafield) [reply DFC.ServiceTaxonomy.Editor/App_Data/Sites/Default/appsettings.template.json]: Didn't remove it intentionally - will ensure it's re-added

File: DFC.ServiceTaxonomy.GraphSync/GraphSyncers/BagPartGraphSyncer.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@
 using OrchardCore.ContentManagement.Metadata.Models;
 using OrchardCore.Flows.Models;
 
-namespace DFC.ServiceTaxonomy.GraphSync.GraphSyncers.Parts
```
Comment (lazcool) [inline DFC.ServiceTaxonomy.GraphSync/GraphSyncers/BagPartGraphSyncer.cs]: Is the file move and namespace change intentional?

File: DFC.ServiceTaxonomy.Neo4j/Log/NeoLogger.cs [hunk 4]
```diff
@@ -19,7 +19,6 @@ public NeoLogger(ILogger<NeoLogger> logger)
 
         public void Info(string message, params object[] args) => _logger.LogInformation(message, args);
 
-        //todo: have setting to selectively log debug lines (e.g. starts with "RUN")
```
Comment (lazcool) [inline DFC.ServiceTaxonomy.Neo4j/Log/NeoLogger.cs]: Looks like this isn't done yet 😄 - looks like a merge issue.

**Pull request closed without merging**